tests/test_web_urldispatcher.py

```diff
@@ -0,0 +1,104 @@
+import pytest
+import tempfile
+import aiohttp
+from aiohttp import web
+import os
+import shutil
+import asyncio
+
+# Timeout in seconds for an asynchronous test:
+ASYNC_TEST_TIMEOUT = 1
+
+
+class ExceptAsyncTestTimeout(Exception):
+    pass
+
+
+def run_timeout(cor, loop, timeout=ASYNC_TEST_TIMEOUT):
```

Comment:
It is used for making sure the test doesn't run too long. Do you have other mechanism to do this in the code base? If not, maybe we can add it in a more global location, to be used with other tests. The ASYNC_TEST_TIMEOUT is the default timeout.